content/blog/2018/07/2018-07-30-introducing-cloud-native-sig.adoc

```diff
@@ -134,7 +134,7 @@ There are many existing plugins allowing to upload and download artifacts from e
 (e.g. S3, Artifactory, Publish over SFTP, etc., etc.),
 but there are no plugins which can do it transparently without using
 new steps.
-In many cases the artifacts also get uploaded through master,
+In many cases the artifacts also get uploaded through the controllers,

```

Comment:
seems ungrammatical

Reply:
Which part? Are you referring to the plural?

Reply:
Yes. This sentence refers to the behavior of some plugins to route uploads from the agents, through the controller, to the final destination. There is only one controller for any given agent.

Reply:
Okay, I will correct this in a subsequent Pr